bundles/AdminBundle/Resources/public/js/pimcore/notification/panel.js

```diff
@@ -0,0 +1,198 @@
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+pimcore.registerNS("pimcore.notification.panel");
+
+pimcore.notification.panel = Class.create({
+
+    initialize: function () {
+        this.getTabPanel();
+    },
+
+    activate: function () {
+        var tabPanel = Ext.getCmp("pimcore_panel_tabs");
+        tabPanel.setActiveItem("pimcore_notification_panel");
+    },
+
+    getTabPanel: function () {
+        if (!this.panel) {
+            var gridPanel = new Ext.Panel({
+                id: 'gridPanel',
+                region: 'center',
+                layout: "fit",
+                items: [
+                    this.getGrid()
+                ]
+            });
+
+            this.panel = new Ext.Panel({
+                id: "pimcore_notification_panel",
+                title: t("Notifications"),
+                iconCls: "pimcore_icon_object",
```

Comment:
there we should also use the correct icon, not the object icon

Reply:
changed, as I wrote above I used comments icon